index.html

```diff
@@ -0,0 +1,205 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>MICROVERSE VIDEO/IMAGE EMBEDED PAGE</title>
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.15.4/css/all.css" integrity="sha384-DyZ88mC6Up2uqS4h/KRgHuoeGwBcD4Ng9SiP4dIRy0EXTlnuz47vAwmeGwVChigm" crossorigin="anonymous"/>
+    <link rel="stylesheet" href="style.css">
+
+</head>
+<body>
+    <!--NAVIGATION SECTION-->
+<div class="navbar">
+    <div class="logo">
+        <span style="font-size:25px;cursor:pointer">&#9776;</span>
+        <p>BIG-LOGO</p>
+    </div>
+
+    <div class="searchField">
+        <input type="text" name="search" id="searchMe" placeholder="Search">
+        <button class="searhway"><i class="fa fa-search" aria-hidden="true"></i>
+        </button>
+        <button class="voiceway"><i class="fa fa-microphone" aria-hidden="true"></i>
+        </button>
+    </div>
+
+    <div class="icongroup">
```

Comment:
Using an unordered list will be great for styling and positioning.
It is preferable in project building